docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -140,46 +143,114 @@ If compiler auto-detection fails for some reason, specify it using
 
 .. code-block::
 
    export <VARIABLE_NAME>=<PATH_TO_MODULE>
    setenv <VARIABLE_NAME> <PATH_TO_MODULE>
 
+.. _MacDetails:
+
+Additional Details for Building on MacOS
+------------------------------------------
+
+.. note::
+    Users not building the SRW App to run on MacOS may skip to the :ref:`next section <BuildExecutables>`. 
+
+The SRW App can be built on MacOS systems, presuming HPC-Stack has already been successfully installed. The following two options have been tested:
+
+* **Option 1:** MacBookAir 2020, M1 chip (arm64, running natively), 4+4 cores, Big Sur 11.6.4, GNU compiler suite v.11.2.0_3 (gcc, gfortran, g++); no MPI pre-installed
+
+* **Option 2:** MacBook Pro 2015, 2.8 GHz Quad-Core Intel Core i7 (x86_64), Catalina OS X 10.15.7, GNU compiler suite v.11.2.0_3 (gcc, gfortran, g++); no MPI pre-installed
+
+.. note::
+    Examples in this subsection presume that the user is running Terminal.app with a bash shell environment. If this is not the case, users will need to adjust the commands to fit their command line application and shell environment. 
+
+The ``build_macos_gnu`` modulefile initializes the module environment, lists the location of HPC-Stack modules, loads the meta-modules and modules, and sets compilers, additional flags, and environment variables needed for building the SRW App. The modulefile must be modified to include the absolute path to the user's HPC-Stack installation and ``ufs-srweather-app`` directories. In particular, the following section must be modified:
+
+.. code-block:: console
+
+   # This path should point to your HPCstack installation directory
+   setenv HPCstack "/Users/username/hpc-stack/install"
+
+   # This path should point to your SRW Application directory
+   setenv SRW "/Users/username/ufs-srweather-app"
+   
+An excerpt of the ``build_macos_gnu`` contents appears below for Option 1. To use Option 2, the user will need to comment out the lines specific to Option 1 and uncomment the lines specific to Option 2 in the ``build_macos_gnu`` modulefile. Additionally, users need to verify that all file paths reflect their system's configuration and that the correct version numbers for software libraries appear in the modulefile. 
+
+.. code-block:: console
+
+   # Option 1 compiler paths: 
+   setenv CC "/opt/homebrew/bin/gcc"
+   setenv FC "/opt/homebrew/bin/gfortran"
+   setenv CXX "/opt/homebrew/bin/g++"
+
+   # Option 2 compiler paths:
+   #setenv CC "/usr/local/bin/gcc"
+   #setenv FC "/usr/local/bin/gfortran"
+   #setenv CXX "/usr/local/bin/g++"
+
+Then, users must source the Lmod setup file, just as they would on other systems, and load the modulefiles needed for building and running SRW App:
+
+.. code-block:: console
+
+   source etc/lmod-setup.sh
```

Comment:
This should be `source etc/lmod-setup.sh macos` instead assuming bash login shell which is most likely the case on a mac.